index.js

```diff
@@ -8,23 +8,24 @@ const core = require('@actions/core')
 
 module.exports = app => {
   app.on('push', async context => {
+    let exitEarly = false
     const config = await getConfig({
       app,
       context,
       getConfig: require('probot-config')
+    }).catch(error => {
+      log({ app, context, error, message: 'Invalid config file' })
+      exitEarly = true
     })
 
+    if (exitEarly) return

```

Comment:
not sure this is the best way to exit early.

Would you prefer try and catch block, problem is the config object is accessed below.

Reply:
Sure thing